app/views/users/_form.html.erb

```diff
@@ -1,9 +1,15 @@
 <%= render 'shared/errors', obj: @user %>
     <%= form_for(@user, :html => {class: "form-horizontal", role: "form"})  do |f| %>
+
+  <div class="form-group">
+    <%= f.select(:role, [['reader', 'reader'], ['author', 'author'], ['Admin', 'admin']]) %>
```

Comment:
We need to remove the admin option from the dropdown, can't let users make themselves admin 😄 Only Reader and Author need to be there. Admin we will do in another feature.

Reply:
hahahaha